interpreter/cling/lib/Interpreter/ClingPragmas.cpp

```diff
@@ -22,19 +22,32 @@ using namespace cling;
 using namespace clang;
 
 namespace {
-  class PHLoad: public PragmaHandler {
-    Interpreter& m_Interp;
+   static void replaceEnvVars(std::string &Path){
 
-  public:
-    PHLoad(Interpreter& interp):
-      PragmaHandler("load"), m_Interp(interp) {}
+    std::size_t fpos = Path.find("$");
 
-    void HandlePragma(Preprocessor &PP,
-                      PragmaIntroducerKind Introducer,
-                      Token &FirstToken) override {
-      // TODO: use Diagnostics!
+    while (fpos != std::string::npos) {
+      std::size_t spos = Path.find("/", fpos + 1);
+      std::size_t length;
+
+      if (spos != std::string::npos) // if we found a "/"
+        length = spos - fpos;
+      else // we didn't find any "/"
+        length = Path.length();
 
-      struct SkipToEOD_t {
+      std::string envVar = Path.substr(fpos + 1, length -1); //"HOME"
+      std::string fullPath{getenv(envVar.c_str())};
```

Comment:
Add test for non-existent var name! (No need to find anything, just call with non-existent var name)